dt-backend/src/model/user/dto/user.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class UserDto {
```

Comment:
Usually, there's a verb in here indicating, which action is happening! i.e.: `UserUpdateDto`. Since the UpdateDto, will be different from the `CreateDto` in both fields and field descriptions!